### PR TITLE
Improved javadocs & add source/docs to installs. Ref: #41

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: src/main/java/kx/c.java [hunk 1]
```diff
@@ -1476,7 +1543,7 @@ public Object k() throws KException,IOException,UnsupportedEncodingException{
      * The default implementation discards async messages, responds to sync messages with an error, 
      * otherwise the remote will continue to wait for a response
      * @param c The c object that received the message
-     * @param msgType The type of the message received
+     * @param msgType The type of the message receivedv (0 – async, 1 – sync, 2 – response)
```
Comment (awilson-kx) [inline src/main/java/kx/c.java:1546]: receivedv ->  received

**Review by awilson-kx — CHANGES_REQUESTED** [review]
Small typo

**Comment by sshanks-kx** [comment]
Typo fixed

**Review by mshimizu-kx — APPROVED** [review]
Confirmed typo fix

**Pull request merged**